server/src/interfaces/search.interface.ts

```diff
@@ -114,7 +114,7 @@ export interface SearchPeopleOptions {
 }
 
 export interface SearchOrderOptions {
-  orderDirection?: 'ASC' | 'DESC';
+  orderDirection?: 'ASC' | 'DESC' | 'RAND';
```

Comment:
I think an API along the lines of `order: { direction: 'ASC' | 'DESC', field: 'fileCreatedAt' | 'random' } would be nicer and more extensible with other sort options in the future.

Reply:
I didn't want to make sweeping changes as this would seem to change quite a few things. Any suggestions about how to go forward with this? Thanks

Reply:
You can change references and types for `orderDirection: <dir>` to `order: { direction: <dir> }` style and have the `field` property be specific to `searchMetadata`.